lib/morse_translator.rb

```diff
@@ -1,43 +1,23 @@
+require './lib/dictionary'
 class MorseTranslator
-
+  attr_reader :english_morse
   def initialize
-    @dictionary = {"a" => ".-",
-                    "b" => "-...",
-                    "c" => "-.-.",
-                    "d" => "-..",
-                    "e" => ".",
-                    "f" => "..-.",
-                    "g" => "--.",
-                    "h" => "....",
-                    "i" => "..",
-                    "j" => ".---",
-                    "k" => "-.-",
-                    "l" => ".-..",
-                    "m" => "--",
-                    "n" => "-.",
-                    "o" => "---",
-                    "p" => ".--.",
-                    "q" => "--.-",
-                    "r" => ".-.",
-                    "s" => "...",
-                    "t" => "-",
-                    "u" => "..-",
-                    "v" => "...-",
-                    "w" => ".--",
-                    "x" => "-..-",
-                    "y" => "-.--",
-                    "z" => "--..",
-                    "1" => ".----",
-                    "2" => "..---",
-                    "3" => "...--",
-                    "4" => "....-",
-                    "5" => ".....",
-                    "6" => "-....",
-                    "7" => "--...",
-                    "8" => "---..",
-                    "9" => "----.",
-                    "0" => "-----",
-                    " " => " "}
+    @english_morse = []
```

Comment:
I'm not crazy about these as instance variables - I would prefer these to be local variables that just live in the translation methods